docs/mathematics/count-of-digits.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: Count of Digits
+slug: /mathematics/count-of-digits
+---
+
+## Introduction
+
+Count of digits refers to the sum of the frequency of all distinct digits in a number.
+
+## Explanation 
+
+We will make a cnt variable and initilize it to `0`. We will keep on dividing the number by `10` and keep on increasing the cnt variable until the number becomes `0`.
```

Comment:
Explanation shouldn't be same as algorithm.

Reply:
done